### PR TITLE
Take credit

### PR DESCRIPTION
**Pull request by JoriDuguid2**

### CODE REVIEW

**Review by growlf — COMMENTED** [review]
Ordinarily, one does not remove the original author from the credits.  I recommend adding your names instead.

File: docker-compose-g6.yml
Comment (growlf) [inline docker-compose-g6.yml:1]: I am not sure that I understand why you would want to map the docker socket to a named volume - much less one name portainerVolume, which makes me suspect that you may be misunderstanding the purpose of the docker.sock system socket file.

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

**Comment by growlf** [comment]
This change has no value or purpose to the intent of this project.

**Pull request closed without merging**